agents/src/multimodal/multimodal_agent.ts

```diff
@@ -178,10 +178,26 @@ export class MultimodalAgent extends EventEmitter {
         this.#speaking = true;
       };
 
-      // eslint-disable-next-line @typescript-eslint/no-unused-vars
       const onPlayoutStopped = (interrupted: boolean) => {
         this.emit('agent_stopped_speaking');
         this.#speaking = false;
+        if (this.#playingHandle) {
+          let text = this.#playingHandle.transcriptionFwd.text;
+          if (interrupted) {
+            text += '…';
```

Comment:
see below - i think you need to change the definition of text to correctly get text that's truncated at the point of interruption

Reply:
in general i couldn't get it to emit interrupted events; interruptions work but it just swallows the attempted event and starts a new one without throwing interrupt

Reply:
i'll tackle this in a later PR, for now non-interrupt works great and i'd rather work on this later